docs/core/containers/sdk-publish.md

```diff
@@ -0,0 +1,200 @@
+            ├── DotNet.ContainerImage.csproj.nuget.g.targets
+            ├── project.assets.json
+            └── project.nuget.cache
+```
+
+The `dotnet new` command creates a new folder named _Worker_ and generates a worker service that, when run, logs a message every second. From your terminal session, change directories and navigate into the _Worker_ folder. Use the `dotnet run` command to start the app.
+
+```dotnetcli
+dotnet run
+Using launch settings from ./Worker/Properties/launchSettings.json...
+Building...
+info: DotNet.ContainerImage.Worker[0]
+      Worker running at: 01/06/2025 13:37:28 -06:00
+info: Microsoft.Hosting.Lifetime[0]
+      Application started. Press Ctrl+C to shut down.
+info: Microsoft.Hosting.Lifetime[0]
+      Hosting environment: Development
+info: Microsoft.Hosting.Lifetime[0]
+      Content root path: .\Worker
+info: DotNet.ContainerImage.Worker[0]
+      Worker running at: 01/06/2025 13:37:29 -06:00
+info: DotNet.ContainerImage.Worker[0]
+      Worker running at: 01/06/2025 13:37:30 -06:00
+info: Microsoft.Hosting.Lifetime[0]
+      Application is shutting down...
+```
+
+The worker template loops indefinitely. Use the cancel command <kbd>Ctrl+C</kbd> to stop it.
+
+## Set the container image name
+
+There are various configuration options available when publishing an app as a container. By default, the container image name is the `AssemblyName` of the project. If that name is invalid as a container image name, you can override it by specifying a `ContainerRepository` as shown in the following project file:
+
+:::code language="xml" source="snippets/Worker/DotNet.ContainerImage.csproj" highlight="8":::
+
+For further reference, see [`ContainerRepository`](publish-configuration.md#containerrepository).
+
+## Publish .NET app
+
+To publish the .NET app as a container, use the following [dotnet publish](../tools/dotnet-publish.md) command:
+
+```dotnetcli
+dotnet publish --os linux --arch x64 /t:PublishContainer
+```
+
+The preceding .NET CLI command publishes the app as a container:
+
+- Targeting Linux as the OS (`--os linux`).
+- Specifying an x64 architecture (`--arch x64`).
+
+> [!IMPORTANT]
+> To publish the container locally, you must have an active OCI-compliant daemon running. If it isn't running when you attempt to publish the app as a container, you experience an error similar to the following:
+>
+> ```console
+> ..\build\Microsoft.NET.Build.Containers.targets(66,9): error MSB4018:
+>    The "CreateNewImage" task failed unexpectedly. [..\Worker\DotNet.ContainerImage.csproj]
+> ```
+
+The `dotnet publish` command produces output similar to the example output:
+
+```dotnetcli
+Restore complete (0.2s)
+  DotNet.ContainerImage succeeded (2.6s) → bin\Release\net9.0\linux-x64\publish\
+```
+
+This command compiles your worker app to the _publish_ folder and pushes the container image to your local Docker daemon by default. If you're using Podman, an alias
```

Comment:
@IEvangelist This sentence is incomplete. What did you mean to say? See #44983.

Reply:
Oops, good catch. I can't recall what I was going to say, @baronfel - in this context, is an Podman alias already present or does the user need to create one?

Reply:
podman will be present in one of two ways:
* standalone podman executable
* podman executable + docker command alias (shadowing)

in both cases our tooling can discover and transparently use them. So maybe the sentence is good enough saying "If you're using Podman, the tooling also supports pushing there with no additional configuration"?